engine/runtime/src/test/java/org/enso/compiler/ParseStdLibTest.java

```diff
@@ -0,0 +1,310 @@
+package org.enso.compiler;
+
+import com.oracle.truffle.api.source.Source;
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.FileVisitResult;
+import java.nio.file.FileVisitor;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.util.function.Function;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+import org.enso.compiler.codegen.AstToIr;
+import org.enso.compiler.core.IR;
+import org.enso.syntax.text.AST.ASTOf;
+import org.enso.syntax.text.Parser;
+import org.enso.syntax.text.Shape;
+import org.junit.runner.RunWith;
+import org.junit.runners.AllTests;
+
+@RunWith(AllTests.class)
+public final class ParseStdLibTest extends TestCase {
+    private static final EnsoCompiler ensoCompiler = new EnsoCompiler();
+    private final File where;
+
+    private ParseStdLibTest(String name, File where) {
+        super(name);
+        this.where = where;
+    }
+
+    public static TestSuite suite() throws Exception {
+        TestSuite s = new TestSuite();
+        collectDistribution(s, "Base");
+        return s;
+    }
+
+    private static Path locateDistribution(final String name) throws URISyntaxException {
+        var where = new File(ParseStdLibTest.class.getProtectionDomain().getCodeSource().getLocation().toURI());
+        var dir = where;
+        for (;;) {
+            dir = new File(new File(new File(new File(new File(new File(where, "distribution"), "lib"), "Standard"), name), "0.0.0-dev"), "src");
```

Comment:
It's 2022 and one can still not write `dir / a / b / c` in Java (but can in Scala!) :)